docs/dev/tutorials.md

```diff
@@ -1,16 +1,16 @@
 # Secret Tutorials
 
-Ramp up on Secret Network apps and contracts development starting from reference tutorials and code repositories.
+Provided are references to aid in building Secret Network apps and smart contract development, beginning with reference tutorials and code repositories.
```

Comment:
References to help build Secret Networks dApps and smart contracts are found below, beginning with reference tutorials and code repositories. 

- [ ] Update copy.